app/bundles/CoreBundle/Assets/js/4.builder.js

```diff
@@ -219,7 +219,7 @@ Mautic.formatCode = function() {
  */
 Mautic.openMediaManager = function() {
     Mautic.openServerBrowser(
-        mauticBasePath + (mauticEnv === 'dev' ? '/index_dev.php' : '') + '/elfinder',
```

Comment:
I see here some changes what not related to this PR

Reply:
@kuzmany this change was made by you. Shall I revert it?

Reply:
Yes